composer.json

```diff
@@ -31,7 +31,7 @@
         "gregwar/captcha": "^1.1",
         "geoip2/geoip2": "~2.0",
         "openclerk/country-list": "^1.0",
-        "mockery/mockery": "^0.9.4",
+        "mockery/mockery": "^1.0",
```

Comment:
Hi @carlos-mg89, I believe that in mockery 1.4 it drops support for PHP < 7.3, so I think this should be >= 1.0 and < 1.4. Maybe I'm missing something.

Thanks!

Reply:
Yes, that's right. I've pointed this out in my notes (GoogleDrive). However, Composer won't let you update if PHP isn't 7.3. You can try by running composer update.

But yes, we can certainly restrict this in the composer.json. I'd do it right away ;)

Reply:
True, I was missing that part!